Geometry_Engine/Query/ClosestPoint.cs

```diff
@@ -33,9 +37,13 @@ public static partial class Query
         /**** Public Methods - Vectors                  ****/
         /***************************************************/
 
+        [Description("Finds the closest point in the provided set of points in relation to the provided Point.")]
+        [Input("cloud", "The set of points from which to find the closest to the provided point.")]
+        [Input("point", "The reference point. The point in the cloud with the smallest distance to this point will be returned.")]
+        [Output("closePt", "The point in the cloud with the smallest distance to the provided point.")]
```

Comment:
Refers to all methods in this file

```suggestion
        [Output("closestPt", "The point in the cloud with the smallest distance to the provided point.")]
```